tests/integration/acp.go

```diff
@@ -110,6 +111,12 @@ func addPolicyACP(
 
 		expectedErrorRaised := AssertError(s.t, s.testCase.Description, err, action.ExpectedError)
 		assertExpectedErrorRaised(s.t, s.testCase.Description, action.ExpectedError, expectedErrorRaised)
+
+		// The policy should only be added to a SourceHub chain once - there is no need to loop through
```

Comment:
question: What happens if a node tries to re-add the same exact policy ? Guessing it will output a new policyID and still add the policy?

Reply:
That has not changed, the test framework will still attempt to re-add the policy.